README.md

```diff
@@ -1,132 +1,38 @@
-Chitter Challenge
-=================
+# Chitter
 
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+A twitter clone for the week 4 end of unit exercise at Makers.
 
-Challenge:
--------
+## Note
 
-As usual please start by forking this repo.
+This code currently has an issue with logins. All unit and feature tests pass, but when trying it yourself, it will not let you log in.
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+To use the site currently, register and do not log out!
 
-Features:
--------
+## Installation
 
+```shell
```

Comment:
Nice job including the setup commands! One minor amendment to add: I had to change `dan` in `database.yml` to my username (:

Reply:
Yes good call! For the next challenge I'm going to look into using Rake files instead.